distro/runtime/concierge/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+DIRNAME=`dirname "$0"`
+PROGNAME=`basename "$0"`
+
+RUNTIME_FOLDER=`cd "$DIRNAME/.."; pwd`
+BASE_FOLDER=`cd "$RUNTIME_FOLDER/.."; pwd`
+
+# set ports for HTTP(S) serverputt
+if [ "x$HTTP_PORT" = "x" ]; then
+    HTTP_PORT="8080"
+fi
+
+if [ "x$HTTPS_PORT" = "x" ]; then
+    HTTPS_PORT="8443"
+fi
+
+# Find the concierge framework jar
+MAIN=$(find framework -name "org.eclipse.concierge-5.0.0*.jar" | sort | tail -1);
+
```

Comment:
O.k. there is now a check on the version. If a JDK smaller than 8 is used the following will displayed:

```
Using java: Q:\software\jdk1.7.0_75
java version "1.7.0_75"
Java(TM) SE Runtime Environment (build 1.7.0_75-b13)
Java HotSpot(TM) 64-Bit Server VM (build 24.75-b04, mixed mode)
ERROR: JVM must be greater than 1.7
```